assets/js/base/components/cart-checkout/shipping-rates-control-package/index.tsx

```diff
@@ -6,10 +6,14 @@ import { _n, sprintf } from '@wordpress/i18n';
 import { decodeEntities } from '@wordpress/html-entities';
 import type { ReactElement } from 'react';
 import type { PackageRateOption } from '@woocommerce/type-defs/shipping';
-import { Panel } from '@woocommerce/blocks-checkout';
+import {
+	Panel,
+	__experimentalApplyCheckoutFilter,
+} from '@woocommerce/blocks-checkout';
 import Label from '@woocommerce/base-components/label';
 import { useSelectShippingRate } from '@woocommerce/base-context/hooks';
 import type { CartShippingPackageShippingRate } from '@woocommerce/type-defs/cart';
+import { sanitize } from 'dompurify';
```

Comment:
Can we install the `@types/dompurify` package as well? You can install it using this command: `npm i --save-dev @types/dompurify`. This will fix the ESLint error and add TS support!

Reply:
Thanks for suggesting this. I just added `@types/dompurify` to add TS support for `dompurify`.